src/Client.php

```diff
@@ -352,8 +352,8 @@ public function request(string $method, string $uri, array $body = [], array $qu
       $response = $this->guzzleClient->request($method, $uri, $requestOptions);
     }
     catch (RequestException $e) {
-      // If the exception is a 'not found' response to a GET, just return false.
-      if ($method === 'GET' && $e->getCode() === 404) {
+      // If the exception is a 'not found' response to a GET or DELETE, just return false.
+      if (($method === 'GET' || $method === 'DELETE') && $e->getCode() === 404) {
```

Comment:
It's certainly no worse than the arguably incorrect pattern we already have in place. :ship: